src/stunuri.c

```diff
@@ -71,21 +67,44 @@ int stunuri_decode(struct stun_uri **sup, const struct pl *pl)
 		return err;
 	}
 
-	if (0 == pl_strcasecmp(&uri.scheme, "stun"))
+	err = stunuri_decode_uri(sup, &uri);
+
+	return err;
+}
+
+/**
+ * Decode a STUN uri from URI object
+ *
+ * @param sup Pointer to allocated STUN uri
+ * @param uri URI object
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int stunuri_decode_uri(struct stun_uri **sup, const struct uri *uri) {
```

Comment:
curly brackets for functions should be on the line after function declaration, like this:

```
function
{
}
```

Reply:
[9f7cac4](https://github.com/baresip/baresip/pull/1960/commits/9f7cac47f4edc12a50dd95faf60441eb601e5509)